sources/PPU/PPU.hpp

```diff
@@ -6,17 +6,18 @@
 #define COMSQUARE_PPU_HPP
 
 #include <cstdint>
-#include "../Memory/AMemory.hpp"
-#include "../Memory/MemoryBus.hpp"
-#include "../Renderer/IRenderer.hpp"
-#include "../Ram/Ram.hpp"
-#include "../Models/Vector2.hpp"
+#include "Memory/AMemory.hpp"
+#include "Memory/MemoryBus.hpp"
+#include "Renderer/IRenderer.hpp"
+#include "Ram/Ram.hpp"
+#include "Models/Vector2.hpp"
 #include "Background.hpp"
 #include "PPUUtils.hpp"
+#include "Debugger/TileViewer/RAMTileRenderer.hpp"
 
 #define FALLTHROUGH __attribute__((fallthrough));
 
-// TODO check if it usefull to have defines instead of constepxr
+// TODO check if it useful to have defines instead of constexpr
```

Comment:
Spoiler: it is not. We should have constexpr

Reply:
fixed